tests/posix/mutex/src/posix_mutex.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <errno.h>
+#include <pthread.h>
+
+#define STACK_SIZE 1024
+
+static K_THREAD_STACK_DEFINE(stack, STACK_SIZE);
+
+pthread_mutex_t mutex1;
+
+void *foo(void *p1)
+{
+	pthread_mutex_lock(&mutex1);
```

Comment:
you are simply using the API's without testing anything. you can create synchronization scenario b/w main thread and child thread.